modules/composer-service-api/src/main/java/org/opencastproject/composer/api/EncodingProfileImpl.java

```diff
@@ -89,7 +89,7 @@
   protected HashMap<String,String> suffixes = new HashMap<String, String>();
 
   @XmlElement(name = "jobLoad")
-  protected Float jobLoad = 1.0f;
+  protected Float jobLoad = 0.6f;
```

Comment:
Is the choice of 0.6f equally as magical as the choice of 1.0f?

Reply:
Yes, it is :) For a default, I think it's ok.

Reply:
To be clear, this change changes *all* encoding to default to 0.6f.  So an 8 core machine with N workflows running now will attempt to process ~13 workflows at once if they're all encoding at the same time.  This seems high, which is why I chose 1.0f initially.  I fully support dropping down the default for the editing step, but I'm curious why we are dropping the cost of all encoding?

Reply:
Hi @gregorydlogan, I just changed because of Lars request to change all the default job loads to < 1 and I tried to follow the relative load among different services (e.g. video editor > encode > inspect, etc). I am fine with changing it to whatever you think is best. For us, we are going to use 4.0 and 8.0 (for multi-profile operations). We have 32-core systems.

Reply:
Fair enough, @lkiesow I can't find the thread where you made that request, what was the reasoning again?